gazebo_plugins/src/gazebo_ros_gpu_laser.cpp

```diff
@@ -34,6 +34,7 @@
 #include <gazebo/sensors/GpuRaySensor.hh>
 #include <gazebo/sensors/SensorTypes.hh>
 #include <gazebo/transport/transport.hh>
+#include <gazebo/gazebo_config.h>
```

Comment:
it might not need it because it should also be included by `gazebo_ros_utils.h`, but I think it doesn't hurt to have it there